pkg/json_raw_results.go

```diff
@@ -193,10 +221,33 @@ type jsonRawResults struct {
 	DefaultBranchCommits []jsonDefaultBranchCommit `json:"default-branch-commits"`
 	// Archived status of the repo.
 	ArchivedStatus jsonArchivedStatus `json:"archived"`
+	// Fuzzers.
+	Fuzzers []jsonFuzzer `json:"fuzzers"`
 	// Releases.
 	Releases []jsonRelease `json:"releases"`
 }
 
+func (r *jsonScorecardRawResult) addFuzzingRawResults(fd *checker.FuzzingData) error {
+	r.Results.Fuzzers = []jsonFuzzer{}
+	for _, f := range fd.Fuzzers {
+		fuzzer := jsonFuzzer{
+			// TODO: Job, File, etc.
+		}
+		switch f.Name {
+		case checker.FuzzerNameCIFuzz:
+			fuzzer.Name = fuzzerNameCIFuzz
+		case checker.FuzzerNameOSSFuzz:
+			fuzzer.Name = fuzzerNameOSSFuzz
+		case checker.FuzzerNameGoBuiltin:
+			fuzzer.Name = fuzzerNameGoBuiltin
+		default:
+			return fmt.Errorf("%w: %d", errorInvalidFuzzer, f.Name)
+		}
+		r.Results.Fuzzers = append(r.Results.Fuzzers, fuzzer)
```

Comment:
It's better to have the type as `string` instead of converting here.

Reply:
problem with strings is that we don't get the linter to warn us when we forget a case in a switch statement

Reply:
I was wrong. string enums work well!